components/EventCard/styles.ts

```diff
@@ -1,28 +1,38 @@
 
 import { StyleSheet } from 'react-native'
-import { lightGray, peach, shadowColor} from '../../assets/colors/colors'
+import { lightGray, lightPeach, peach, shadowColor} from '../../assets/colors/colors'
 
 export const styles = StyleSheet.create({
   cardContainer: {
-    backgroundColor: peach,
+    backgroundColor: lightPeach,
+    borderColor: peach,
     borderRadius: 10,
-    elevation: 5,
-    margin: 10,
-    overflow: 'hidden',
+    borderWidth: 2,
+    flex: 1,
+    height: 202,
+    margin: 5,
+    overflow: 'scroll',
+    
     shadowColor: shadowColor,
     shadowOffset: {
       height: 2,
       width: 0,
     },
     shadowOpacity: 0.25,
     shadowRadius: 3.84,
+    width: 172,
```

Comment:
the width should be set to percentage terms, never use constant width parameters. E.g. here we want it to be `95%` of the whole screen width